pom.xml

```diff
@@ -41,6 +41,11 @@
             <artifactId>spring-boot-starter-data-jpa</artifactId>
         </dependency>
 
+        <dependency>
+            <groupId>org.hibernate.validator</groupId>
```

Comment:
In Spring Boot projects, it's usually better to opt for starters rather than individual dependencies (look for spring-boot-starter-validation)